features/user_should_be_able_to_edit_articles.feature

```diff
@@ -0,0 +1,19 @@
+Feature: User should have the ability to edit articles
+  As an publisher/journalist
+  In order to keep the article fresh and up to date
+  I would like to be able to edit an article
+
+  Background:
+    Given the following articles exist
+      | title | body |
+      | Jade loves cookies | Brownies, I love Brownies |
+
+  Scenario: User edits a article
+    Given I am on the "Jade loves cookies" page
+    When I click on "Edit Article"
+    And I fill in "Title" with "Jade loves SailorCookie"
+    And I fill in "Content" with "Dreamteam for the win!"
+    And I click on "Submit Changes"
+    Then I should be on the "Jade loves SailorCookie" page
+    And I should see "Jade loves SailorCookie"
+    And I should see "Dreamteam for the win!"
```

Comment:
I don't know if you have any validations on your `Article` model (IMO you should at least validate the presence of `#title` and `#body`) so the lack of a sad path is okay. 

However, I see that you use a ternary operator in your controller to check if the article actually updates, so you must have considered that it might NOT updata succesluffy. So a test would be in place.